configs/agent/ci.toml

```diff
@@ -55,6 +55,16 @@ reserved-cpu = 1
 reserved-mem = "1G"
 reserved-disk = "8G"
 
+[logging.graylog]
+host = "localhost"
+port = 12201
+
+# If True, validate the Graylog server’s certificate. In this case specifying ca_certs is also required.
+validate = false
```

Comment:
I'd like to rename this to `ssl-verify` following the convention in the existing `logstash` driver config.